core/autoload/spacevim.vim

```diff
@@ -145,11 +145,18 @@ function! spacevim#end() abort
   call s:config()
   if exists('*UserConfig') | call UserConfig() | endif
 
-  try
-    execute 'helptags' g:spacevim.base . '/core/doc'
-  catch
-    echom v:exception
-  endtry
+  let s:hlptag_file = g:spacevim.base . '/core/doc/spacevim.txt'
```

Comment:
```suggestion
  let s:helptag_file = g:spacevim.base . '/core/doc/spacevim.txt'
```

Reply:
Oh ok I did that.

I shouldn't have used s: on those variables. Fixed that too.